src/main/java/edu/wpi/grip/core/operations/opencv/SizeGeneratorOperation.java

```diff
@@ -0,0 +1,46 @@
+package edu.wpi.grip.core.operations.opencv;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import org.bytedeco.javacpp.opencv_core.Size;
+
+public class SizeGeneratorOperation implements CVOperation {
+
+    private final SocketHint<Number> widthHint = new SocketHint<Number>("width", Number.class, -1, SocketHint.View.SPINNER, new Integer[]{Integer.MIN_VALUE, Integer.MAX_VALUE}, false);
```

Comment:
Should the minimum value be 0?  I don't think negative sizes make sense, and OpenCV will probably spit back a bunch of errors if we try to to that.